index.html

```diff
@@ -18,52 +18,259 @@
-          me.
-        </p>
+    <section class="hero">
+      <h2>
+        I’m Teke<br />Glad to see<br />
+        you!
+      </h2>
+      <p class="paragraph">
+        I’m a software developer! I can help you build a product , feature or
+        website Look through some of my work and experience! If you like what
+        you see and have a project you need coded, don’t hestiate to contact me.
+      </p>
 
-        <span class="connect">LETS CONNECT</span>
+      <span class="connect">LETS CONNECT</span>
 
-        <div class="socialmedia_icons_container">
-          <ul class="media_icons">
-            <li>
-              <a href="#">
-                <img src="images/icons/twitter.svg" alt="Twitter Icon" />
-              </a>
-            </li>
-            <li>
-              <a href="#">
-                <img src="images/icons/linkedin.png" alt="LinkedIn Icon" />
-              </a>
-            </li>
-            <li>
-              <a href="#">
-                <img src="images/icons/m.svg" alt="Media Icon" />
-              </a>
+      <div class="socialmedia_icons_container">
+        <ul class="media_icons">
+          <li>
+            <a href="#">
+              <img src="images/icons/twitter.svg" alt="Twitter Icon" />
+            </a>
+          </li>
+          <li>
+            <a href="#">
+              <img src="images/icons/linkedin.png" alt="LinkedIn Icon" />
+            </a>
+          </li>
+          <li>
+            <a href="#">
+              <img src="images/icons/m.svg" alt="Media Icon" />
+            </a>
+          </li>
+          <li>
+            <a href="#">
+              <img src="images/icons/cat.svg" alt="Github Icon" />
+            </a>
+          </li>
+          <li>
+            <a href="#">
+              <img src="images/icons/fingers.svg" alt="Victory Icon" />
+            </a>
+          </li>
+        </ul>
+      </div>
+    </section>
+
+    <!--Works Section-->
+    <section class="works">
+      <div class="grid-container">
+        <div class="card-layout">
+          <a href="#">
+            <img
+              class="image-fluid"
+              src="images/works/image_1.png"
+              alt="background_image"
+            />
+          </a>
+          <div class="p-title">
+            <h2 class="title">Tonic</h2>
+            <ul class="flex-items">
+              <li class="list-item canopy">CANOPY</li>
+              <li class="list-item role">Back End Dev</li>
+              <li class="list-item year">2015</li>
+            </ul>
+            <p class="text_paragraph">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+            <div class="btn-container-flex">
+              <button type="button" class="btn-sm">html</button>
+              <button type="button" class="btn-md">css</button>
+              <button type="button" class="btn-lg">javaScript</button>
+            </div>
```

Comment:
- [ ] As requested by the previous code reviewer and project requirements, Kindly wrapp the languages into [```ul li```](https://github.com/microverseinc/curriculum-html-css/blob/3e39af5af8075c97c82e7fec70edd3ff40345234/portfolio/2_mobile.md?plain=1#L51-L52) tags. e.g 
```html 
<ul class="btn-container-flex">
  <li type="button" class="btn-sm">html</li>
  <li type="button" class="btn-md">css</li>
  <li type="button" class="btn-lg">javaScript</li>
</ul>
```
then polish up the UI to match the design, should this change affect how awesome your buttons currently look.


- [ ] I would like to mention how well your project looks. You've done an amazing job! 👏🏽 . The only things that need changes as per project specifications are structure requirements and Figma template reference. For example, I can see that the background color here did not match your [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122) for the languages buttons. Kindly make use of the `inspect` section in the Figma app to get the exact color codes. Here is a [guideline article link](https://help.figma.com/hc/en-us/articles/360055203533-Use-the-Inspect-panel) to help you with this one 🤓

Reply:
Thanks, @teke85 .

Kindly note that The code snippet was just an example, for your reference. However, for list items. the `type` attribute is not needed.

Reply:
> * [ ]  As requested by the previous code reviewer and project requirements, Kindly wrapp the languages into [`ul li`](https://github.com/microverseinc/curriculum-html-css/blob/3e39af5af8075c97c82e7fec70edd3ff40345234/portfolio/2_mobile.md?plain=1#L51-L52) tags. e.g
> 
> ```
> <ul class="btn-container-flex">
>   <li type="button" class="btn-sm">html</li>
>   <li type="button" class="btn-md">css</li>
>   <li type="button" class="btn-lg">javaScript</li>
> </ul>
> ```
> 
> then polish up the UI to match the design, should this change affect how awesome your buttons currently look.
> 
> * [ ]  I would like to mention how well your project looks. You've done an amazing job! 👏🏽 . The only things that need changes as per project specifications are structure requirements and Figma template reference. For example, I can see that the background color here did not match your [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122) for the languages buttons. Kindly make use of the `inspect` section in the Figma app to get the exact color codes. Here is a [guideline article link](https://help.figma.com/hc/en-us/articles/360055203533-Use-the-Inspect-panel) to help you with this one 🤓

@blessedjasonmwanza @BertrandConxy I have wrapped the elements up as advised by you and the previous reviewer
![ul li](https://user-images.githubusercontent.com/29442846/177199692-753a4e79-63c0-4937-a8af-8e15e62c4769.png)
.